src/data_scraper/scraper_binance.py

```diff
@@ -51,6 +58,31 @@ def get_data(self, start_time, end_time, overwrite=False):
             else:
                 df = df.merge(temp_df, how='outer', on=config.MERGE_DATA_ON)
 
-        utils.save(data=df, subfolder='binance', filename=f'binance_{self.interval}.csv', overwrite=overwrite)
+        # Creating general datetime column in seconds
+        df["datetime"] = df[f"{currency_pair}_Open time"]
+
+        return df
+
+    def load_dataset(self, start_time, end_time):
+        full_dataset_name = f"{self.dataset_name}_{start_time}_{end_time}"
```

Comment:
I imagine the availability of the data will depend on the naming, i.e. even if I have data from `2021-01-01` to `2021-05-01` (the filename would be sth like `TwitterGeneric_2021_01_01_2021_05_01` but I want data for `2021-01-15` to `2021-05-01` (so I look for `TwitterGeneric_2021_01_15_2021_05_01`, it will start scraping it, although it's available, right?